### PR TITLE
About us

### PR DESCRIPTION
**Pull request by sccarmichael**

### CODE REVIEW

File: src/pages/about/index.js [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+                  <p className={styles.prev}>
+                    Previously:
+                    <br /> Product design manager @ WeWork, Founder of The
+                    UpStanding Desk
+                  </p>
+                  <div className={styles.social}>
+                    <a
+                      target="_blank"
+                      rel="noopener noreferrer"
+                      href="https://linkedin.com/in/sambauch"
```
Comment (sccarmichael) [inline src/pages/about/index.js:128]:
```suggestion
                      href="https://linkedin.com/in/samcarmichael"
```

File: src/pages/about/index.js [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+                    >
+                      <LinkedinFilled />
+                    </a>
+                    <a
+                      target="_blank"
+                      rel="noopener noreferrer"
+                      href="https://linkedin.com/in/sambauch"
```
Comment (sccarmichael) [inline src/pages/about/index.js:135]:
```suggestion
                      href="https://github.com/sccarmichael"
```

File: src/pages/about/index.js [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+                    >
+                      <GithubFilled />
+                    </a>
+
+                    <a
+                      target="_blank"
+                      rel="noopener noreferrer"
+                      href="https://linkedin.com/in/sambauch"
```
Comment (sccarmichael) [inline src/pages/about/index.js:143]:
```suggestion
                      href="https://twitter.com/sccarmichael"
```

File: src/pages/about/index.js [hunk 4]
```diff
@@ -0,0 +1,255 @@
...
+                    >
+                      <TwitterSquareFilled />
+                    </a>
+                    <a
+                      target="_blank"
+                      rel="noopener noreferrer"
+                      href="https://linkedin.com/in/sambauch"
```
Comment (sccarmichael) [inline src/pages/about/index.js:150]:
```suggestion
                      href="https://sam-carmichael.com"
```

**Comment by sccarmichael** [comment]
@sbauch this is looking great - one request, is it possible to set a max width (of like... 375px?) on the team pics? the reason i ask is that if you shrink the window they get funky, i think mine is higher res so it is way bigger than yours and MT's

File: src/pages/about/index.js [hunk 5]
```diff
@@ -0,0 +1,247 @@
...
+                    </a>
+
+                    <a
+                      target="_blank"
+                      rel="noopener noreferrer"
+                      href="https://linkedin.com/in/sambauch"
+                    >
+                      <TwitterSquareFilled />
+                    </a>
+                  </div>
```
Comment (sbauch) [inline src/pages/about/index.js:190]: @mike6838 could you fill in here in this suggestion block your social links, and remove whatever you dont want to include?
```suggestion
                  <div className={styles.social}>
                    <a
                      target="_blank"
                      rel="noopener noreferrer"
                      href="https://linkedin.com/in/sambauch"
                    >
                      <LinkedinFilled />
                    </a>
                    <a
                      target="_blank"
                      rel="noopener noreferrer"
                      href="https://linkedin.com/in/sambauch"
                    >
                      <GithubFilled />
                    </a>

                    <a
                      target="_blank"
                      rel="noopener noreferrer"
                      href="https://linkedin.com/in/sambauch"
                    >
                      <TwitterSquareFilled />
                    </a>
                  </div>
```

File: src/pages/about/index.js [hunk 6]
```diff
@@ -0,0 +1,247 @@
...
+                    display: "flex",
+                    flexGrow: "1",
+                    flexDirection: "column",
+                    padding: "16px 0 0",
+                  }}
+                >
+                  <Card.Meta title="Sam Carmichael" description="Design" />
+                  <p className={styles.prev}>
+                    Previously:
+                    <br /> Product design manager @ WeWork, Founder of The
```
Comment (sccarmichael) [inline src/pages/about/index.js:121]:
```suggestion
                    <br /> Product design @ WeWork, Founder of The
```

**Review by sbauch — APPROVED** [review]
im good with this. yall?

**Pull request merged**